classes/GmailTransport.php

```diff
@@ -81,12 +82,43 @@ private static function base64url(Swift_Mime_SimpleMessage $message) {
      */
     public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = null) {
         try {
-            $encodedMessage = $this::base64url($message);
+            // Check if the total mail mime size is less than 5 MB
+            if(strlen($message->toString()) < 5 * 1024 * 1024) {
```

Comment:
This `if` statement can be removed, as the non-chunked sending method becomes redundant, leaving the chunked sending method which can handle any size (up to the max 35MB).